### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by vadimpylsky**

### CODE REVIEW

File: src/ios/ExportOptions.plist [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>compileBitcode</key>
-	<true/>
-	<key>destination</key>
-	<string>export</string>
-	<key>method</key>
-	<string>enterprise</string>
-	<key>provisioningProfiles</key>
```
Comment (wcoder) [inline src/ios/ExportOptions.plist:11]: This file is required for the release build. Please revert it.

File: src/pubspec.yaml [hunk 2]
```diff
@@ -92,7 +88,6 @@ flutter:
   assets:
     - assets/icons/
 
-# https://marketplace.visualstudio.com/items?itemName=localizely.flutter-intl
```
Comment (wcoder) [inline src/pubspec.yaml:95]: It's a helpful link about used VScode extension. Please revert it.

File: src/ios/.gitignore [hunk 3]
```diff
@@ -1,32 +0,0 @@
-*.mode1v3
-*.mode2v3
-*.moved-aside
-*.pbxuser
```
Comment (wcoder) [inline src/ios/.gitignore:4]: It's a helpful ios gitignore file. Please revert it.

File: src/ios/ExportOptions.plist (hunk 1 above)
Comment (vadimpylsky) [reply src/ios/ExportOptions.plist:11]: done

File: src/ios/.gitignore (hunk 3 above)
Comment (vadimpylsky) [reply src/ios/.gitignore:4]: strange I think this file needed for submodules,
but ok... I will return it

**Review by wcoder — APPROVED** [review]

**Pull request merged**